index.html

```diff
@@ -1910,39 +1919,44 @@ <h3>Add Proof</h3>
       <section>
         <h3>Add Proof Set/Chain</h3>
         <p>
-          The following algorithm specifies how to incrementally add a proof to
+          The following algorithm specifies how to incrementally add a proof to 
           a proof set or proof chain starting with a secured document containing
           either a proof or proof set/chain.
-          Required inputs are a <a>secured data document</a>
-          (<var>securedDocument</var>) and <a>proof options</a>
+          Required inputs are a <a>secured data document</a> 
+          (<var>securedDocument</var>) and <a>proof options</a> 
           (<var>options</var>). The <a>proof options</a>
-          (<var>options</var>) must satisfy the criteria in
+          (<var>options</var>) must satisfy the criteria in 
           section <a href="#add-proof"></a>.
-          A new <a>secured data document</a> is produced as
-          output. Whenever this algorithm encodes strings, it MUST use UTF-8
+          A new <dfn>secured data document</dfn> is produced as
+          output. Whenever this algorithm encodes strings, it MUST use UTF-8 
           encoding.
         </p>
 
         <ol class="algorithm">
           <li>
-Let <var>proof</var> be set to <var>securedDocument</var>.<var>proof</var>. Let
-<var>proof_list</var> be an empty list. If <var>proof</var> is a list copy all
-the elements of <var>proof</var> to <var>proof_list</var>. If <var>proof</var>
+Let <var>proof</var> be set to <var>securedDocument</var>.<var>proof</var>. Let 
+<var>proof_list</var> be an empty list. If <var>proof</var> is a list copy all 
+the elements of <var>proof</var> to <var>proof_list</var>. If <var>proof</var> 
 is an object add a copy of that object to <var>proof_list</var>.
           </li>
           <li>
-Let the <var>unsecuredDocument</var> be a copy of the <var>securedDocument</var>
-with the <var>proof</var> attribute removed. Let <var>output</var> be a copy of
+Let the <var>unsecuredDocument</var> be a copy of the <var>securedDocument</var> 
+with the <var>proof</var> attribute removed. Let <var>output</var> be a copy of 
```

Comment:
Strip trailing whitespace for these lines (and all lines below that have trailing whitespace). It makes it harder to read PRs because it makes it seem like more has changed in the PR than actually has changed.